soh/src/overlays/actors/ovl_En_Kakasi2/z_en_kakasi2.c

```diff
@@ -118,9 +118,10 @@ void func_80A90264(EnKakasi2* this, GlobalContext* globalCtx) {
 
     this->unk_194++;
 
-    if ((BREG(1) != 0) && (this->actor.xzDistToPlayer < this->maxSpawnDistance.x) &&
-        (fabsf(player->actor.world.pos.y - this->actor.world.pos.y) < this->maxSpawnDistance.y)) {
-
+    if (((BREG(1) != 0) && (this->actor.xzDistToPlayer < this->maxSpawnDistance.x) &&
+            (fabsf(player->actor.world.pos.y - this->actor.world.pos.y) < this->maxSpawnDistance.y)) ||
```

Comment:
did a quick little reformat to try to better see the logic in this if statement (not recommending this as a code change)
```c

if (
    (
     (BREG(1) != 0) &&
     (this->actor.xzDistToPlayer < this->maxSpawnDistance.x) &&
     (fabsf(player->actor.world.pos.y - this->actor.world.pos.y) < this->maxSpawnDistance.y)
    ) ||
    (
     (CVar_GetS32("gSkipScarecrow",0) != 0) &&
     (globalCtx->msgCtx.ocarinaAction == OCARINA_ACTION_FREE_PLAY) &&
     gSaveContext.scarecrowSpawnSongSet
    )
   )
```
it seems like the logic here is skipping all the distance/position checks when skip scarecrow is enabled. is that intentional?

Reply:
Oops, it definitely is. I meant to include the positional logic as an && to it, but I left it out for debugging purposes. I'll add that back in!